README.md

```diff
@@ -97,8 +97,7 @@ To ensure you followed the steps correctly, check https://telemetry.polkadot.io/
 
 * [Aura](https://github.com/paritytech/substrate/tree/master/srml/aura)
 * [Balances](https://github.com/paritytech/substrate/tree/master/srml/balances)
-* [Consensus](https://github.com/paritytech/substrate/tree/master/srml/consensus)
-* [Contract](https://github.com/paritytech/substrate/tree/master/srml/contract)
+* [Contracts](https://github.com/paritytech/substrate/tree/master/srml/contracts)
 * [Council](https://github.com/paritytech/substrate/tree/master/srml/council)
```

Comment:
We need to update the README for the new modules.

Reply:
Right right